src/time/time.h

```diff
@@ -239,7 +239,7 @@ static inline proc_time_i
 time_memcache2proc_sec(memcache_time_u t)
 {
     if (t == 0) {
-        return INT32_MAX - 1;
+        return INT32_MAX;
```

Comment:
For consistency should we use `INT64_MAX` instead of `INT64_MAX - 1` in the finer grain timestamps?

Reply:
yes, thanks for reminding me. done.